### DISCUSSION TITLE
How can I start server with unix domain socket like Nuxt2 on Nuxt3?

### DISCUSSION THREAD

**Discussion by yoshihir0** [nuxt/nuxt#23771] [Questions]
Hi,

I want start nuxt3 server with socket (not port) like nuxt2.

### nuxt2 nuxt.config.js
```
export default {
  server: {
    socket: '/tmp/nuxt.socket'   <- like this
  }
}
```

**Comment by thebladehit** [comment]
I have the same problem.

I fixed it like this:

I built project and in .output/server/chunks/runtime.mjs found **port** const on 5787 line.
I changed port const to:
![image](https://github.com/nuxt/nuxt/assets/65012653/cb648b6c-24d2-4a17-8f87-e10fd59a8cdd)
Then I found where it is used, below in the code is **listener** const(on 5790 line):
![image](https://github.com/nuxt/nuxt/assets/65012653/f71c852f-3a76-42e7-9a9a-6aa193decdc5)
Then changed like this:
![image](https://github.com/nuxt/nuxt/assets/65012653/06760c1f-ee67-45f3-b534-591d4d4558b6)

After all this, run code:
![image](https://github.com/nuxt/nuxt/assets/65012653/27eeda9e-0872-40df-a80a-34cf16a2d708)
Then I had message about starting server on unix server...
